gammapy/stats/fit_statistics.py

```diff
@@ -4,13 +4,13 @@
 see :ref:`fit-statistics`
 """
 import numpy as np
+from gammapy.stats.fit_statistics_cython import TRUNCATION_VALUE
 
 __all__ = ["cash", "cstat", "wstat", "get_wstat_mu_bkg", "get_wstat_gof_terms"]
 
 N_ON_MIN = 1e-25
```

Comment:
I just noticed that here we already define a truncation value n_on and cstat. Might be good to merge both...but this could be done in a follow up PR as well...

Reply:
It is done